[T2A3][T2A4][F10] Tan Boon Joon

tanboonjoon opened the pull request:
[T2A3] Decrease SLAP and compare results and [T2A4]Smelliest code ever!

mauris commented:
@tanboonjoon which tutorial are you attending? Do follow the instructions to submit separate pull requests from separate branches per activity. If unsure, take a look at how your classmates are doing it!

tanboonjoon commented:
Added my tutorial class,
Both activity 3 and 4 are under [LO-MethodAbstraction], am i not allowed to combined them? 

removed SLAP on processProgramArgs method
inline executeCommand method
ready for review
